src/components/controls/keyboard-controls.js

```diff
@@ -0,0 +1,94 @@
+var registerControls = require('../../core/controls').registerControls;
```

Comment:
Also, slight risk that by removing `keyboard-controls` here we'll encourage more people to keep using the now-deprecated `wasd-controls` + `look-controls` combination. Maybe that's avoidable with strong warnings.

We could also get going on a quick follow-up PR for `hotspot` and `hotspot-controls` components.